### ISSUE TITLE
L1000 landmark genes are different in CDRP comparing to TAORF, LUAD and LINCS

### ISSUE THREAD

**Issue by MarziehHaghighi** [broadinstitute/2018_04_20_Rosetta#18]
- CDRP landmark set seems to be different, there are 785 prob ids in common:

![Screen Shot 2022-02-23 at 7 46 28 PM](https://user-images.githubusercontent.com/30931779/155435544-13f99aa2-224c-4e3b-af7c-1416db34d32e.png)

- If we use this file (/home/ubuntu/bucket/projects/2018_04_20_Rosetta/workspace/metadata/affy_probe_gene_mapping.txt) to convert the prob ids to gene symbols an then check the overlap, the results wont change:
![Screen Shot 2022-02-23 at 7 55 21 PM](https://user-images.githubusercontent.com/30931779/155436376-f27f746d-7f9f-493d-9300-8a152bdcbcb2.png)

- **191 Genes that are just in CDRP and not the rest of datasets:** 
<details>

|     | Just-In-CDRP          |
|----:|:----------------------|
|   0 | TGDS                  |
|   1 | TRIP10                |
|   2 | WWOX                  |
|   3 | HBE1                  |
|   4 | CTSL2                 |
|   5 | BTG2                  |
|   6 | HIST1H2BD             |
|   7 | BTBD3                 |
|   8 | FANCA                 |
|   9 | HLF                   |
|  10 | ASL                   |
|  11 | DTNA                  |
|  12 | SLC16A6               |
|  13 | ZBTB5                 |
|  14 | DOK4                  |
|  15 | FTSJ1                 |
|  16 | GGA2                  |
|  17 | ASAP2                 |
|  18 | CREB3L2               |
|  19 | BCKDHA                |
|  20 | BXDC5                 |
|  21 | IFIT5                 |
|  22 | PARN                  |
|  23 | SLC36A1               |
|  24 | PPP2R5B               |
|  25 | SERTAD3               |
|  26 | IL1RAP                |
|  27 | TMEM127               |
|  28 | PTPLA                 |
|  29 | ABAT                  |
|  30 | FANCL                 |
|  31 | QRSL1                 |
|  32 | UGP2                  |
|  33 | KDELR3                |
|  34 | TOMM22                |
|  35 | UBE4A                 |
|  36 | ALMS1                 |
|  37 | PPAP2A                |
|  38 | KIAA0922              |
|  39 | DHPS                  |
|  40 | ERLIN1                |
|  41 | SEC61A2               |
|  42 | ZNF768                |
|  43 | SETD1B                |
|  44 | CTCF                  |
|  45 | RAD23B                |
|  46 | WRN                   |
|  47 | KHDC1///SPA17         |
|  48 | HMGCL                 |
|  49 | FDX1                  |
|  50 | SLC2A1                |
|  51 | PPP4R1                |
|  52 | KIAA0317              |
|  53 | ZFP161                |
|  54 | GRHPR                 |
|  55 | RAB1B                 |
|  56 | AFF1                  |
|  57 | CMPK1                 |
|  58 | ZGPAT                 |
|  59 | TTF1                  |
|  60 | EIF1B                 |
|  61 | SEC24B                |
|  62 | FAM162A               |
|  63 | RAP1A                 |
|  64 | MARK4                 |
|  65 | VPS26A                |
|  66 | SMYD3                 |
|  67 | PEX19                 |
|  68 | GPRC5C                |
|  69 | PANK2                 |
|  70 | PEPD                  |
|  71 | BTG3                  |
|  72 | NSFL1C                |
|  73 | EPHB4                 |
|  74 | SRF                   |
|  75 | SWAP70                |
|  76 | GPR125                |
|  77 | LYPD3                 |
|  78 | PJA1                  |
|  79 | CTNS                  |
|  80 | SETDB1                |
|  81 | PARL                  |
|  82 | MAD2L1BP              |
|  83 | ANKRD49               |
|  84 | ECHDC1                |
|  85 | MPDZ                  |
|  86 | LASP1                 |
|  87 | PER2                  |
|  88 | CHCHD7                |
|  89 | TCIRG1                |
|  90 | BAZ2B                 |
|  91 | HADHB                 |
|  92 | TBCB                  |
|  93 | COQ3                  |
|  94 | FZD5                  |
|  95 | MGST2                 |
|  96 | TLE3                  |
|  97 | FUZ                   |
|  98 | TNFAIP1               |
|  99 | PHF2                  |
| 100 | SNRPD1                |
| 101 | CEP55                 |
| 102 | CD19                  |
| 103 | IMPA2                 |
| 104 | ACSL1                 |
| 105 | ZC3H4                 |
| 106 | HDAC4                 |
| 107 | TSN                   |
| 108 | ERCC5                 |
| 109 | AKAP1                 |
| 110 | NAT13                 |
| 111 | ERCC6L                |
| 112 | GM2A                  |
| 113 | ASMTL                 |
| 114 | ARHGEF5               |
| 115 | BAZ1B                 |
| 116 | DSE                   |
| 117 | TP53TG1               |
| 118 | SFRS16                |
| 119 | CISH                  |
| 120 | CD14                  |
| 121 | PDS5B                 |
| 122 | SFMBT1                |
| 123 | CSDA                  |
| 124 | SIK1                  |
| 125 | F12                   |
| 126 | STAT6                 |
| 127 | CA12                  |
| 128 | UVRAG                 |
| 129 | MAPKSP1               |
| 130 | CLCN3                 |
| 131 | EPHA2                 |
| 132 | ADCY9                 |
| 133 | SEC24C                |
| 134 | FAM3C                 |
| 135 | LRRC40                |
| 136 | RABGGTA               |
| 137 | PTEN                  |
| 138 | GYS1                  |
| 139 | ZNF692                |
| 140 | SFRS2B                |
| 141 | PSMD5                 |
| 142 | COL4A5                |
| 143 | ZER1                  |
| 144 | DLGAP5                |
| 145 | DFNB31                |
| 146 | FADD                  |
| 147 | LHPP                  |
| 148 | SAP18                 |
| 149 | RARA                  |
| 150 | EPB41L4B              |
| 151 | SENP5                 |
| 152 | DNAJB12               |
| 153 | CABIN1                |
| 154 | SNRPF                 |
| 155 | FEZ1                  |
| 156 | FBRS                  |
| 157 | ENTPD6                |
| 158 | DHX8                  |
| 159 | GMDS                  |
| 160 | TARS                  |
| 161 | ZNF629                |
| 162 | RAB3GAP1              |
| 163 | EFNB3                 |
| 164 | CKB                   |
| 165 | LIMK2                 |
| 166 | DALRD3///LOC100133719 |
| 167 | MAN2A2                |
| 168 | PKD1                  |
| 169 | VAMP7                 |
| 170 | SRPRB                 |
| 171 | ETFA                  |
| 172 | PPP2R3A               |
| 173 | SRRM1                 |
| 174 | ERCC1                 |
| 175 | PPP3CA                |
| 176 | DDX49                 |
| 177 | GOLIM4                |
| 178 | PPIF                  |
| 179 | SGK1                  |
| 180 | USP20                 |
| 181 | MEGF9                 |
| 182 | SNX2                  |
| 183 | SMG7                  |
| 184 | NIPBL                 |
| 185 | SEC14L1               |
| 186 | RTCD1                 |
| 187 | RHOBTB1               |
| 188 | ZBTB24                |
| 189 | ZNF672                |
| 190 | POLR3B                |
| 191 | RBM16                 |

</details>

- **192 Genes that are in LUAD, TAORF and LINCS and NOT in CDRP:**

<details>

|     | Just-In-LUAD-LINCS-TAORF   |
|----:|:---------------------------|
|   0 | SRC                        |
|   1 | HLA-DMA                    |
|   2 | AXIN1                      |
|   3 | SERPINE1                   |
|   4 | RALGDS                     |
|   5 | APP                        |
|   6 | ETS1                       |
|   7 | SATB1                      |
|   8 | TLR4                       |
|   9 | MMP2                       |
|  10 | MEF2C                      |
|  11 | CCNE2                      |
|  12 | ADRB2                      |
|  13 | RHEB                       |
|  14 | CASP2                      |
|  15 | CRK                        |
|  16 | CYCS                       |
|  17 | FOXO3                      |
|  18 | COL1A1                     |
|  19 | PPARD                      |
|  20 | FZD1                       |
|  21 | ATF1                       |
|  22 | HMOX1                      |
|  23 | PDGFA                      |
|  24 | EGR1                       |
|  25 | PSMD9                      |
|  26 | CDKN1B                     |
|  27 | TGFB3                      |
|  28 | PTPRC                      |
|  29 | PTGS2                      |
|  30 | PCNA                       |
|  31 | PIK3R4                     |
|  32 | SNCA                       |
|  33 | TGFBR2                     |
|  34 | MAT2A                      |
|  35 | PSMD2                      |
|  36 | CASP7                      |
|  37 | PARP1                      |
|  38 | BAX                        |
|  39 | PSMD4                      |
|  40 | NOS3                       |
|  41 | BRCA1                      |
|  42 | MAPKAPK2                   |
|  43 | HSPD1                      |
|  44 | DAXX                       |
|  45 | BIRC2                      |
|  46 | RHOA                       |
|  47 | CDK6                       |
|  48 | FRS2                       |
|  49 | EIF4G1                     |
|  50 | PIK3CA                     |
|  51 | ALDOA                      |
|  52 | AURKA                      |
|  53 | NFKBIA                     |
|  54 | PSMB8                      |
|  55 | CBLB                       |
|  56 | ASAH1                      |
|  57 | PROS1                      |
|  58 | PRKX                       |
|  59 | DAG1                       |
|  60 | PSME1                      |
|  61 | MNAT1                      |
|  62 | HLA-DRA                    |
|  63 | HSPA1A///HSPA1B            |
|  64 | PLCB3                      |
|  65 | ARHGEF12                   |
|  66 | ICAM1                      |
|  67 | FOSL1                      |
|  68 | CSNK1E                     |
|  69 | GLI2                       |
|  70 | GNAI1                      |
|  71 | CASP10                     |
|  72 | HSPB1                      |
|  73 | CCND1                      |
|  74 | IGF2BP2                    |
|  75 | IGFBP3                     |
|  76 | FBXO11                     |
|  77 | MKNK1                      |
|  78 | CSNK1A1                    |
|  79 | DFFB                       |
|  80 | FYN                        |
|  81 | RPS6                       |
|  82 | ATF6                       |
|  83 | HMGA2                      |
|  84 | RPS5                       |
|  85 | FGFR4                      |
|  86 | PSMB10                     |
|  87 | CHP                        |
|  88 | HIF1A                      |
|  89 | PIK3R3                     |
|  90 | SPP1                       |
|  91 | CDC42                      |
|  92 | EGF                        |
|  93 | CALM1///CALM2///CALM3      |
|  94 | NFATC3                     |
|  95 | NFATC4                     |
|  96 | PSMF1                      |
|  97 | EIF4EBP1                   |
|  98 | GNA15                      |
|  99 | C5                         |
| 100 | FGFR2                      |
| 101 | DFFA                       |
| 102 | SYK                        |
| 103 | CDK5R1                     |
| 104 | TICAM1                     |
| 105 | PTPN6                      |
| 106 | CDK7                       |
| 107 | CLTC                       |
| 108 | CEBPA                      |
| 109 | BNIP3L                     |
| 110 | NCK2                       |
| 111 | SUV39H1                    |
| 112 | PRKCH                      |
| 113 | MYL9                       |
| 114 | STAT5B                     |
| 115 | EDN1                       |
| 116 | AURKB                      |
| 117 | RALA                       |
| 118 | CDC20                      |
| 119 | CDC25B                     |
| 120 | CXCL2                      |
| 121 | DUSP3                      |
| 122 | GRB7                       |
| 123 | CCNA1                      |
| 124 | MYLK                       |
| 125 | PSME2                      |
| 126 | SUZ12                      |
| 127 | EED                        |
| 128 | DUSP6                      |
| 129 | SIRT3                      |
| 130 | SNAP25                     |
| 131 | PTK2B                      |
| 132 | SOX2                       |
| 133 | IFNAR1                     |
| 134 | IL1B                       |
| 135 | RB1                        |
| 136 | E2F2                       |
| 137 | NFKBIB                     |
| 138 | NCK1                       |
| 139 | EPHB2                      |
| 140 | BTK                        |
| 141 | TFDP1                      |
| 142 | IKZF1                      |
| 143 | CDC25A                     |
| 144 | EPHA3                      |
| 145 | PIK3C3                     |
| 146 | NFKBIE                     |
| 147 | DUSP4                      |
| 148 | PGAM1                      |
| 149 | CCNH                       |
| 150 | MUC1                       |
| 151 | MMP1                       |
| 152 | CDK4                       |
| 153 | RASA1                      |
| 154 | PLK1                       |
| 155 | SPTAN1                     |
| 156 | SHC1                       |
| 157 | SQSTM1                     |
| 158 | PPP1R13B                   |
| 159 | ETV1                       |
| 160 | HSPA8                      |
| 161 | TP53                       |
| 162 | NFKB2                      |
| 163 | SKP1                       |
| 164 | RAC2                       |
| 165 | PAK1                       |
| 166 | ERBB3                      |
| 167 | DNMT3A                     |
| 168 | TERT                       |
| 169 | RALB                       |
| 170 | CCL2                       |
| 171 | CDKN2A                     |
| 172 | NOTCH1                     |
| 173 | GAPDH                      |
| 174 | BMP4                       |
| 175 | ERO1L                      |
| 176 | NRAS                       |
| 177 | APOE                       |
| 178 | PRKCQ                      |
| 179 | CD40                       |
| 180 | PAK6                       |
| 181 | PFKL                       |
| 182 | STAT3                      |
| 183 | CHEK1                      |
| 184 | CCNB1                      |
| 185 | GHR                        |
| 186 | CCND3                      |
| 187 | TBP                        |
| 188 | PRKAG2                     |
| 189 | PLA2G4A                    |
| 190 | LOC284889///MIF            |
| 191 | COL4A1                     |
| 192 | FZD7                       | 

</details>

**Comment by shntnu** [comment]
A few quick notes

- The CDRP landmarks seem to correspond to the 2011 patent https://patentimages.storage.googleapis.com/67/c0/b9/2aa36bbe6fa20d/WO2011127150A2.pdf
- The other landmarks seem to correspond to the 2017 paper https://www.cell.com/cms/10.1016/j.cell.2017.10.049/attachment/2f7013d4-01e5-4027-9264-031da5b830d7/mmc1.xlsx (Table S2 L1000 Probes) as well as https://clue.io/command?q=/gene-space%20lm

**Comment by shntnu** [comment]
(We will address this is the paper; stub below)

The L1000 landmark genes in the CDRP dataset are different from the landmark genes in the other datasets, with an overlap of n=785 (80%). The CDRP dataset was acquired using the so-called “delta prime” probe set. Subsequent datasets (LUAD, TAORF, LINCS) were acquired using the so-called “epsilon” probe set [cite https://clue.io/connectopedia/guide_to_geo_l1000_data].

The 20% of the delta prime landmark genes that are absent in epsilon can be inferred using the epsilon landmark genes [https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5990023/]. 

To simplify our analysis, we did not perform this inference, and instead only used the landmark genes available for each dataset. When combining CDRP with other datasets, we used the intersection of the two probe sets.
